src/promptflow-tools/tests/constants.py

```diff
@@ -0,0 +1,304 @@
+            {"location": "San Francisco, CA", "temperature": "72", "unit": null}
+
+            # tool:
+            ## tool_call_id:
+            call_003
+            ## content:
+            {"location": "San Francisco, CA", "temperature": "72", "unit": null}
+        """,
+        [
+            {
+                "role": "tool",
+                "tool_call_id": "call_001",
+                "content": '{"location": "San Francisco, CA", "temperature": "72", "unit": null}',
+            },
+            {
+                "role": "tool",
+                "tool_call_id": "call_002",
+                "content": '{"location": "San Francisco, CA", "temperature": "72", "unit": null}',
+            },
+            {
+                "role": "tool",
+                "tool_call_id": "call_003",
+                "content": '{"location": "San Francisco, CA", "temperature": "72", "unit": null}',
+            },
+        ],
+        True,
+        "",
+        "",
+    ),
+    (
+        """
+            # assistant:
+            ## tool_calls:
+            [{'id': 'call_001', 'function': {'arguments': '{"location": {"city": "San Francisco", "country": "USA"}, "unit": "metric"}', 'name': 'get_current_weather_py'}, 'type': 'function'}, {'id': 'call_002', 'function': {'arguments': '{"location": {"city": "Tokyo", "country": "Japan"}, "unit": "metric"}', 'name': 'get_current_weather_py'}, 'type': 'function'}, {'id': 'call_003', 'function': {'arguments': '{"location": {"city": "Paris", "country": "France"}, "unit": "metric"}', 'name': 'get_current_weather_py'}, 'type': 'function'}]
+        """,
+        [
+            {
+                "role": "assistant",
+                "tool_calls": [
+                    {
+                        "id": "call_001",
+                        "function": {
+                            "arguments": '{"location": {"city": "San Francisco", "country": "USA"}, "unit": "metric"}',
+                            "name": "get_current_weather_py",
+                        },
+                        "type": "function",
+                    },
+                    {
+                        "id": "call_002",
+                        "function": {
+                            "arguments": '{"location": {"city": "Tokyo", "country": "Japan"}, "unit": "metric"}',
+                            "name": "get_current_weather_py",
+                        },
+                        "type": "function",
+                    },
+                    {
+                        "id": "call_003",
+                        "function": {
+                            "arguments": '{"location": {"city": "Paris", "country": "France"}, "unit": "metric"}',
+                            "name": "get_current_weather_py",
+                        },
+                        "type": "function",
+                    },
+                ],
+            }
+        ],
+        True,
+        "",
+        "",
+    ),
+    (
+        """
+            # system:
```

Comment:
Can we save it in jinja file and load in conftest? Then future tes cases can reuse the test prompt in the future?

![image](https://github.com/microsoft/promptflow/assets/49483542/285c0d47-2cb1-4f3c-9958-4548d834669f)

Reply:
moved